### PR TITLE
[EventHub] Bump uAMQP Dependency to v1.2.13 

### PR DESCRIPTION
**Pull request by yunhaoling**

### CODE REVIEW

File: sdk/eventhub/tests.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ jobs:
     parameters:
       AllocateResourceGroup: 'false'
       ServiceDirectory: eventhub
+      InjectedPackages: $(WheelLocationsList)
```
Comment (yunhaoling) [inline sdk/eventhub/tests.yml:8]:
```suggestion
```

**Comment by yunhaoling** [comment]
python - eventhub - tests

File: sdk/eventhub/azure-eventhub/CHANGELOG.md [hunk 2]
```diff
@@ -2,9 +2,15 @@
...
+**Bug fixes**
 
-* Updated uAMQP dependency to 1.2.12 adding support for Python 3.9.
+* Updated `azure.eventhub.extension.__init__.py` to be compatible with pkgutil-style namespace (PR #13210, thanks @pjachowi).
+* Updated uAMQP dependency to 1.2.13
+  - Added support for Python 3.9.
+  - Fixed bug that macOS was unable to detect network error (#15473).
+  - Fixed bug that `uamqp.ReceiveClient` and `uamqp.ReceiveClientAsync` receive messages during connection establishment (#15555).
+  - Fixed bug where connection establishment on macOS with Clang 12 triggering unrecognized selector exception (#15567).
+  - Fixed bug in accessing message properties triggering segmentation fault when the underlying C bytes are NULL (#15568).
```
Comment (yunhaoling) [inline sdk/eventhub/azure-eventhub/CHANGELOG.md:13]: although those issues are from ServiceBus not from EventHub, but as both of the SDKs share the uamqp dependency, I think mentioning the issues bring some value for those who use both SDKs as they might incur the same problem with EH.

but if it's not suitable to mention another SDK issues, I would be glad to remove the relevant issues.

**Review by annatisch — APPROVED** [review]

**Review by rakshith91 — APPROVED** [review]

**Pull request merged**